module/definition.nix

```diff
@@ -107,7 +121,7 @@ let
       };
 
       perms = mkOption {
-        description = "The mode of the secret file.";
+        description = "The octal mode of the secret file.";
```

Comment:
Maybe note that this should be a string, containing a four-digit integer, and that the leading digit should almost always be zero? Not sure.

Reply:
Cole mentioned on a call that Vault might imply a zero, it might be useful to check this and write the conclusion in the docs for the option.